docs/cmake-project-view.md

```diff
@@ -0,0 +1,124 @@
+# CMake Options Visibility Configuration
+
+Starting in CMake Tools 1.16, we will provide users the ability to customize their status bar with related CMake items like presets, kits, variants, build, configure, and launch to allow for users to de-clutter this global status bar as much as they would like to, while also having quick access to commonly performed CMake actions.
+
+Users can specify what is shown in the status bar through their `settings.json` file.
+
+The default settings will be set to the following:
+## Default Settings Json
+```json
+"cmake.options.statusBarVisibility": "hidden"
+"cmake.options.advanced": {
+    "build": {
+        "statusBarVisibility": "visible"
+    },
+    "launch": {
+        "statusBarVisibility": "visible"
+    },
+    "debug": {
+        "statusBarVisibility": "visible"
+    }
+}
+```
+
+These settings mean that by default all settings will be hidden from the status bar, except for Build, Launch, and Debug actions.
+
+To revert to the prior experience and have all of your presets display in the status bar, set your `Cmake > Options: StatusBar: Visibility` setting to `Visible`. This setting is overwritten by`CMake > Options: Advanced setting`, so if you had added to have some options hidden through this, these options will still remain hidden. 
```

Comment:
the first setting is `Cmake > Options: Status Bar Visibility` and the second shouldn't have the word "setting" within the ``.
Fixed version:
To revert to the prior experience and have all of your presets display in the status bar, set your `Cmake > Options: Status Bar Visibility` setting to `Visible`. This setting is overwritten by `Cmake > Options: Advanced` setting, so if you had added to have some options hidden through this, these options will still remain hidden.